hplsql/src/test/java/org/apache/hive/hplsql/TestHplsqlLocal.java

```diff
@@ -31,10 +27,7 @@
  */
 public class TestHplsqlLocal {
 
-  private final ByteArrayOutputStream out = new ByteArrayOutputStream();
-  private final ByteArrayOutputStream err = new ByteArrayOutputStream();
-
-  @Test
+    @Test
```

Comment:
Please fix indentation. @test should have same ident as the method.

Reply:
Fixed in https://github.com/apache/hive/pull/5381/commits/e96c827734d75bb87dc7be3354e63a2a285adefe